flake.nix

```diff
@@ -84,6 +84,27 @@
             '';
           };
         };
+
+      } // rec {
+        python38 = pkgs.python310.override {
+          sourceVersion = {
+            major = "3";
+            minor = "8";
+            patch = "18";
+            suffix = "";
+          };
+          hash = "sha256-P/txzTSaMmunsvrcfn34a6V33ZxJF+UqhAGtvadAXj8=";
+        };
+
+
+        python38Full = python38.override {
+          self = python38Full;
+          pythonAttr = "python38Full";
+          bluezSupport = true;
```

Comment:
why are we even bothering to build bluetooth support in a repl?

Reply:
It's bluez for audio support I think, not bluetooth. I anycase I just copied from upstrteam.